modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/NavigationButtons.js

```diff
@@ -23,25 +23,32 @@ const NavigationButtons = ({moveController, position}) => {
 				right: position.right,
 				top: position.top,
 			}}
+			type="button"
 		>
-			<button className="box top" onClick={() => moveController('up')}>
-				<ClayIcon className="icon" symbol="angle-left" />
-			</button>
-			<button
-				className="box right"
-				onClick={() => moveController('right')}
+			<div
```

Comment:
Hey @garofalog, why are you using a div instead of a button? If it's something related to styles I think you can try use an `<a type="button"` there :)

Reply:
sorry. <a type="button"> works fine.
Can i keep going on this pr? i opened some subtask to work about what we said yet. [here](https://issues.liferay.com/browse/COMMERCE-6910), [here](https://issues.liferay.com/browse/COMMERCE-6907), [here](https://issues.liferay.com/browse/COMMERCE-6906) and [https://issues.liferay.com/browse/COMMERCE-6905](url) and [here](https://issues.liferay.com/browse/COMMERCE-6909)